itest/loadtest/send_test.go

```diff
@@ -29,30 +30,181 @@ func sendTest(t *testing.T, ctx context.Context, cfg *Config) {
 	ctxt, cancel := context.WithTimeout(ctxb, cfg.TestTimeout)
 	defer cancel()
 
+	sendType := stringToAssetType(cfg.SendAssetType)
+
 	t.Logf("Running send test, sending %d asset(s) of type %v %d times",
-		cfg.NumAssets, cfg.SendType, cfg.NumSends)
+		cfg.NumAssets, sendType, cfg.NumSends)
 	for i := 1; i <= cfg.NumSends; i++ {
 		send, receive, ok := pickSendNode(
-			t, ctx, cfg.NumAssets, cfg.SendType, alice, bob,
+			t, ctx, cfg.NumAssets, sendType, alice, bob,
 		)
 		if !ok {
 			t.Fatalf("Aborting send test at attempt %d of %d as "+
 				"no node has enough balance to send %d "+
 				"assets of type %v", i, cfg.NumSends,
-				cfg.NumAssets, cfg.SendType)
+				cfg.NumAssets, sendType)
 			return
 		}
 
 		sendAssets(
-			t, ctxt, cfg.NumAssets, cfg.SendType, send, receive,
+			t, ctxt, cfg.NumAssets, sendType, send, receive,
 			bitcoinClient, cfg.TestTimeout,
 		)
 
 		t.Logf("Finished %d of %d send operations", i, cfg.NumSends)
 	}
 }
 
-// sendAssets sends the given number of assets of the given type from the given
+// sendTestV2 checks that we are able to send assets between the two nodes. It
+// is a more performant and lightweight version of sendTest, as it uses less
+// assertions and RPC calls.
+func sendTestV2(t *testing.T, ctx context.Context, cfg *Config) {
+	// Start by initializing all our client connections.
+	alice, bob, bitcoinClient := initClients(t, ctx, cfg)
+
+	ctxb := context.Background()
+	ctxt, cancel := context.WithTimeout(ctxb, cfg.TestTimeout)
+	defer cancel()
+
+	sendType := stringToAssetType(cfg.SendAssetType)
+
+	// Alice is set to be the minter in mintV2, so we use Alice's universe.
+	var (
+		uniHost = fmt.Sprintf(
+			"%s:%d", alice.cfg.Host, alice.cfg.Port,
+		)
+	)
+
+	// Let's make sure Bob is aware of all the assets that Alice may have
+	// minted.
+	itest.SyncUniverses(ctx, t, bob, alice, uniHost, cfg.TestTimeout)
```

Comment:
I think we probably have to add a new parameter to this function that allows us to sync transfer proofs as well.
When I ran this test a second time after running 1 mintV2 and 1 sendV2 test, this sync never completed, because it only tried syncing issuance proofs.